fili-core/src/test/groovy/com/yahoo/bard/webservice/metadata/DataSourceMetadataLoadTaskSpec.groovy

```diff
@@ -46,109 +70,73 @@ class DataSourceMetadataLoadTaskSpec extends BaseDataSourceMetadataSpec {
         ]
     }
 
-    List<String> dimensions13 = [dimensions[0], dimensions[2]]*.asName()
-    List<String> quotedDimensions = dimensions13.collect() { '"' + it + '"'}
-
-    List<String> metrics123 = [metrics[0], metrics[1], metrics[2]]*.asName()
-    List<String> metrics13 = [metrics[0], metrics[2]]*.asName()
-
-    def generateSegment(tableName, interval, version, dimensions, metrics, partitionNum, partitions, binVersion, size) {
-        return """{
-                        "dataSource": "$tableName",
-                        "interval": "$interval",
-                        "version": "$version",
-                        "loadSpec": { },
-                        "dimensions": "$dimensions",
-                        "metrics": "$metrics",
-                        "shardSpec": {
-                            "type": "hashed",
-                            "partitionNum": $partitionNum,
-                            "partitions": $partitions
-                        },
-                        "binaryVersion": $binVersion,
-                        "size": $size,
-                        "identifier": ""
-        }"""
-    }
+    def setup() {
+        dimensions13 = [
+                dimensions.(TestApiDimensionName.BREED.asName()).asName(),
```

Comment:
I didn't know you could do that (wrap `TestApiDimensionName.BREED.asName()` in parenthesis to use the dot syntax, rather than the `[]` syntax).

Reply:
@archolewa I did that just incase if people change `BREED` API name in the future, they don't break this test 😌  Does that look good to you?